### ISSUE TITLE
protein aligner:  it would be great to have the se...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#6104]
*Help Desk query collected when no chat operators were online. Follow up required.*

protein aligner:  it would be great to have the sequence numbers for each protein in the alignment at the end of the line.

**Reported by:** paul********** (pws@***********)
**Submitted from:** <a target="_blank" href="http://www.wormbase.org/http://www.wormbase.org/species/c_elegans/protein/WP:CE03840#068359--10">http://www.wormbase.org/species/c_elegans/protein/WP:CE03840#068359--10</a>
**Browser:** Chrome 63.0.3239.132

**Comment by kyook** [comment]
Hey @paulwsternberg and @sibyl229  what is the status of this issue?  I'm not sure I understand the request, is it to include aa number at the end of each line to help with navigating the protein sequence in each species?

**Comment by paulwsternberg** [comment]
Yes, each label on the left shows the total length of the protein.  Having a number at the end of each line would be nice...
